pkg/controller/controller_test.go

```diff
@@ -2456,11 +2459,47 @@ func assertNumEvents(t *testing.T, strings []string, number int) {
 	}
 }
 
+// failfFunc is a type that defines the common signatures of T.Fatalf and
+// T.Errorf.
+type failfFunc func(t *testing.T, msg string, args ...interface{})
+
+func fatalf(t *testing.T, msg string, args ...interface{}) {
+	t.Log(string(debug.Stack()))
+	t.Fatalf(msg, args...)
```

Comment:
@arschles has often discouraged me from doing this-- and for good reason. If you fail within a test helper, the source of the failure will (at first glance) appear to be within the test helper. Assessing the true source of the failure requires a more careful reading of the stack trace than might otherwise be required. I would avoid doing this.